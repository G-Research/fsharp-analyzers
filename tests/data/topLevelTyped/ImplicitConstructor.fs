module IC

type Foo(length) =
    let mutable hashCode = length + 1
