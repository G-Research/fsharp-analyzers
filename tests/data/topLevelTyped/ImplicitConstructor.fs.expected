GRA-UNTYPEDAPI-001 | Warning | (3,5 - 3,8) | `The constructor of Foo` is not private, not fully typed and not used outside `A.fs` in the current project.
Add type annotation for parameter `length`.
